doc/api/fs.md

```diff
@@ -3540,14 +3543,21 @@ If `options` is a string, then it specifies the encoding:
 fs.writeFile('message.txt', 'Hello Node.js', 'utf8', callback);
 ```
 
-Any specified file descriptor has to support writing.
+### File Descriptors
+1. Any specified file descriptor has to support writing.
+2. If a file descriptor is specified as the `file`, it will not be closed
+automatically.
+3. The writing will begin at the beginning of the file. If the file size
+is 10 bytes and if six bytes are written with this file descriptor, then
+`writeFile` will return six bytes newly written and four bytes from the file.
```

Comment:
This seems a bit confusing as `writeFileSync()` returns `undefined` and `writeFile()` transfers only error to the calback. Maybe something like this this?
```suggestion
the file contents would be six bytes newly written and four bytes from the file.
```